wp-content/themes/core/pcss/components/button/_outline.pcss

```diff
@@ -5,7 +5,8 @@
  * ----------------------------------------------------------------------------- */
 
 .c-btn-outline,
-a.c-btn-outline {
+a.c-btn-outline,
+a[data-mce-selected].c-btn-outline {
```

Comment:
What is this `[data-mce-selected]` attribute?

Reply:
@ryanurban Adding this shows the style immediately when a user selects a button style from the formats dropdown in a WYSIWYG editor. Example screenshots:
Without the attribute: http://p.tri.be/ApkUSE
With the attribute: http://p.tri.be/qBM7Vm

Reply:
Still not sure I follow on this. Let's do a quick call.

Reply:
Thanks for the demo, that makes sense!